src/amo/pages/Block/index.js

```diff
@@ -102,6 +102,23 @@ export class BlockBase extends React.Component<InternalProps> {
     return content;
   }
 
+  renderVersion() {
```

Comment:
Can you use the plural form here? (because we render the min and max versions)

Reply:
No Problem!